profile/README.md

```diff
@@ -1,33 +1,38 @@
+[NVIDIA Holoscan](https://developer.nvidia.com/holoscan-sdk) is the AI sensor processing platform that combines hardware systems for low-latency sensor and network connectivity, optimized libraries for data processing and AI, and core microservices to run streaming, imaging, and other applications, from embedded to edge to cloud. It can be used to build streaming AI pipelines for a variety of domains, including Medical Devices, High Performance Computing at the Edge, Industrial Inspection and more.
 
 > In previous releases, the prefix [`Clara`](https://developer.nvidia.com/industries/healthcare) was used to define Holoscan as a platform designed initially for [medical devices](https://www.nvidia.com/en-us/clara/developer-kits/). As Holoscan has grown, its potential to serve other areas has become apparent. With version 0.4.0 of the Holoscan SDK, we're proud to announce that Holoscan is now officially built to be domain-agnostic and can be used to build sensor AI applications in multiple domains. Going forward, domain specific content will be hosted on the [HoloHub](https://github.com/nvidia-holoscan/holohub) repository.
 
-Holoscan consists of:
+The Holoscan platform consists of:
 
 ## Holoscan Software
-* **Holoscan SDK**
+
+* **Holoscan SDK**: the Development Kit
     * [User guide](https://docs.nvidia.com/clara-holoscan/sdk-user-guide/index.html)
-    * [Source code](https://github.com/nvidia-holoscan/holoscan-sdk)
-    * [Python wheel](https://pypi.org/project/holoscan)
+    * [Development container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan)
     * [Debian package](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_dev_deb)
-    * [Docker container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan)
-* **HoloHub**
+    * [Python wheel](https://pypi.org/project/holoscan)
+    * [Source code](https://github.com/nvidia-holoscan/holoscan-sdk)
+* **HoloHub**: community-driven collection of Holoscan applications, operators, and tutorials
     * [Source code](https://github.com/nvidia-holoscan/holohub)
-* **Deployment Software Stack**
-    * [Holoscan OpenEmbedded/Yocto recipes](https://github.com/nvidia-holoscan/meta-tegra-holoscan)
-    * [Holoscan OpenEmbedded/Yocto build container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan-oe-builder)
+* **Deployment Software Stack**: OpenEmbedded/Yocto layer for optimized deployment of the Holoscan SDK on the Developer Kits
+    * [Build container](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/containers/holoscan-oe-builder)
+    * [Source code](https://github.com/nvidia-holoscan/meta-tegra-holoscan)
```

Comment:
I would keep recipes, or at least "source code and recipes". 
Just saying source code doesn't indicate to me at least that it has recipes in it, though admittedly I'm not representative of the target audience.

Reply:
Instead of "Source code", just say "Recipes". There is really no source code in meta-tegra-holoscan, it is a collection of recipes.

Reply:
@mahdiazizian , do you think recipes should come first before the build container?